### ISSUE TITLE
function “display_event_banner” does not support alt text well

### ISSUE THREAD

**Issue by ashokdudhat** [wpeventmanager/wp-event-manager#1425]
Their function for displaying a single image banner (“display_event_banner”) has an issue where if you don’t have an organizer, no alt text of any kind appears. Rather than making use of the built-in attachment functions (e.g. wp_get_attachment_image), and the respective meta data (like ALT text data), they’ve rolled their own, and it sucks.

For all single-image versions of the banner image, their function calls for an organizer to fill in the alt data:
alt="' . esc_attr(get_organizer_name($post)) . '"

Of course, if you have no organizer for an event (and this is OFTEN the case), there is no fallback mechanism. None of this would be an issue, if they just used the built-in functions.
